agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/instrumentation/sdk/TelemetryClientClassFileTransformer.java

```diff
@@ -722,14 +706,14 @@ public static void main(String[] args) {
     @SuppressWarnings("unused")
     public static class TC {
 
-        private TelemetryConfiguration configuration;
+        private com.microsoft.applicationinsights.TelemetryConfiguration configuration;
```

Comment:
just curious to know why removing imports and using explicit types?

Reply:
this is commented out code, wanted to it compile when uncommenting it, without having to add the import statements after uncommenting it